.travis.yml

```diff
@@ -1,3 +1,16 @@
 language: node_js
+cache:
+  directories:
+    - ~/.npm
+    - node_modules
+notifications:
+  email: false
```

Comment:
Don't know if you'd rather want to keep this. I mostly just copied my `travis.yml` file.